phpunit/blocks/renderBlockCorePostTitle.php

```diff
@@ -0,0 +1,171 @@
+<?php
+/**
+ * Tests for the gutenberg_render_block_core_post_title() function.
+ *
+ * @package WordPress
+ * @subpackage Blocks
+ *
+ * @covers ::gutenberg_render_block_core_post_title
+ * @group blocks
+ */
+class Tests_Blocks_RenderBlockCorePostTitle extends WP_UnitTestCase {
+	/**
+	 * Post object.
+	 *
+	 * @var array
+	 */
+	protected static $post;
+
+	/**
+	 * Array of attributes.
+	 *
+	 * @var int
+	 */
+	protected static $attributes;
+
+	/**
+	 * Block object.
+	 *
+	 * @var object
+	 */
+	protected static $block;
+
+	/**
+	 * Setup method.
+	 *
+	 * @param WP_UnitTest_Factory $factory Helper that lets us create fake data.
+	 */
+	public static function wpSetUpBeforeClass( WP_UnitTest_Factory $factory ) {
+		self::$post = $factory->post->create_and_get( array( 'post_title' => 'Post title block Unit Test' ) );
+
+		self::$attributes = array(
+			'textAlign'  => '',
+			'level'      => 2,
+			'isLink'     => false,
+			'linkTarget' => '',
+			'rel'        => '',
+		);
+
+		self::$block = new stdClass();
+
+		$block_args = array(
+			'blockName'    => 'core/post-title',
+			'attrs'        => array(
+				'textColor' => 'red',
+			),
+			'innerBlock'   => array(),
+			'innerContent' => array(),
+			'innerHTML'    => array(),
+		);
+
+		WP_Block_Supports::init();
+		WP_Block_Supports::$block_to_render = $block_args;
+	}
+
+	/**
+	 * Tear down method.
+	 */
+	public static function wpTearDownAfterClass() {
+		wp_delete_post( self::$post->ID, true );
+	}
+
+	/**
+	 * Test gutenberg_render_block_core_post_title() method.
+	 */
+	public function test_should_render_empty_string_when_title_is_empty() {
+
+		// call render method with block context.
+		$rendered = gutenberg_render_block_core_post_title( self::$attributes, '', self::$block );
+		$this->assertEmpty( $rendered );
+
+		self::$block->context = array( 'postId' => 0 );
+		$rendered             = gutenberg_render_block_core_post_title( self::$attributes, '', self::$block );
+		$this->assertEmpty( $rendered );
+	}
+
+	/**
+	 * Test gutenberg_render_block_core_post_title() method.
```

Comment:
`gutenberg_render_block_core_post_title` is not a method.
```suggestion
	 * Test if the function returns correct titles.
```